shopping-cart/shopping-cart-java/README.md

```diff
@@ -1,46 +1,60 @@
 # Shopping Cart
 
-This sample application demonstrates a simple shopping cart built with Lagom. It contains two services, a shopping cart service, for managing shopping carts, and an inventory service, for tracking inventory.
+This sample application demonstrates a simple shopping cart built with Lagom. It contains two services, a shopping cart service, for managing shopping carts, and a inventory service, for tracking inventory.
 
-The shopping cart service persists its data to a relational database using [Akka Persistence Typed](https://doc.akka.io/docs/akka/2.6/typed/persistence.html) API, and is intended to demonstrate how to persist state using Lagom.
+The **shopping cart** service persists its data to a relational database using Lagom's persistence API and demonstrates how to persist state using Lagom.
 
-The inventory service consumes a stream of events published to Kafka by the shopping cart service, and is intended to demonstrate how Kafka event streams can be consumed in Lagom. However, it doesn't persist its state to a database, it just stores it in memory, and this memory is not shared across nodes. Hence, it should not be used as an example of how to store state in Lagom.
+The **inventory service** consumes a stream of events published to Kafka by the shopping cart service, and demonstrates how to consume Kafka event streams in Lagom. However, it doesn't persist its state to a database, it just stores it in memory, and this memory is not shared across nodes. Hence, it should not be used as an example of how to persist state in Lagom.
 
-## Setup
+## Requirements
 
-To run this application locally you will need access to a Postgres database. We suggest you run it on a docker container but a local or remote native instance will also work.
+This sample requires Kafka and Postgres as external services. They are pre-configured in `docker-compose.yml` file. Meaning that before running the service, you first need to start the services using the following command:
 
-We provide a `docker-compose.yml` file that you can use to run a Postgres database already configured for this application. The docker container will be exposed on port `5432`.
-
-To create the image and start the container, run the command below at the root of this project.
-
-```bash 
+```bash
 docker-compose up -d
```

Comment:
I found this confusing in the past. Maybe indicate what folder this is supposed to be run on?

Reply:
Add a `cd` command on both README files.